src/components/common/ColonyHome/ColonyFundingWidget/TokenItem.tsx

```diff
@@ -10,13 +10,13 @@ import { Token } from '~types';
 import styles from './ColonyFundingWidget.css';
 
 interface Props {
-  currentDomainId: number;
+  // currentDomainId: number;
   token: Token;
   isTokenNative?: boolean;
   isNativeTokenLocked?: boolean;
 }
 
-const displayName = 'dashboard.ColonyHome.ColonyFundingWidget.TokenItem';
+const displayName = 'common.ColonyHome.ColonyFundingWidget.TokenItem';
```

Comment:
This is not something you did here, it's just something I ran into so I'm leaving it for future reference.

We have two files named `TokenItem`. This one, and another one in `shared/TokenEditDialog/TokenItem`

We should rename one of them to keep things less confusing